### ISSUE TITLE
Option to disable the tab's index

### ISSUE THREAD

**Issue by mroavi** [jose-elias-alvarez/buftabline.nvim#12]
Not sure if this is possible at this moment. I'm trying to disable the tab's index so that only the name is displayed.

I tried the following approach:
```lua
local options = {
	-- index_format = "%d: ",
	index_format = "",
}
require("buftabline").setup(options)
```
but this didn't work. If this is not possible to do now, then it would be a nice feature to have.

**Comment by jose-elias-alvarez** [comment]
Right, the implementation at the moment doesn't allow for fully disabling the index, but it shouldn't be hard to support. 

Not sure if you saw #11, but I've added your suggestions to the to-do list. If you have any more suggestions, let me know – I should have time to sit down and work on this in a couple of weeks.

**Comment by mroavi** [comment]
Thanks Jose. I did see #11 and that's why I suggested a few more potential improvements. Thanks for the plugin.

**Comment by jose-elias-alvarez** [comment]
Closed by #16.
